frontend/src/components/closeButton/CloseButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./CloseButton.module.css";
+import MaskedDiv from "../atoms/masked/MaskedDiv";
+import Button from "../atoms/button/Button";
+import closeButtonImage from "../../assets/images/landing-close-button.svg";
+import cx from "classnames";
+
+type Props = {
+  onClose?: () => void;
+  className?: string;
+};
+
+export default ({ onClose, className }: Props) => {
+  return (
+    <MaskedDiv
+      className={cx(styles.closeButtonDiv, className)}
+      mask={`url(${closeButtonImage})`}
```

Comment:
if there is a circle background on this we can make visible i think that would look better. not super important by maybe it's here and transparent